sdk/build.gradle

```diff
@@ -19,6 +19,7 @@ android {
         buildConfigField "int", "VERSION_CODE", "$omise_sdk_code_version"
         buildConfigField "String", "VERSION_NAME", "\"$omise_sdk_version\""
         buildConfigField "String", "NETCETERA_LICENSE_KEY", "\"${System.getenv('NETCETERA_LICENSE_KEY')}\""
```

Comment:
It seems we are no longer need NETCETERA_LICENSE_KEY, right?

Reply:
yes will remove